test/tools/api.js

```diff
@@ -145,3 +145,844 @@ module.exports.classNames = new Set(api.map(({ className }) => className))
 module.exports.classNameToMethodList = new Map(api.map((api, _, array) =>
   [api.className, sorted(Array.from(new Set(Array.from(array.filter(v => v.className === api.className), method => method))), (a, b) => byStrings(a.method, b.method))]
 ));
+
+/**
+ * Generated from `api`, this is an exhaustive list of all methods we can unit test.
+ */
+module.exports.unitTestableAPI = [
+  {
+    "className": "Admin",
+    "method": "buildInfo",
```

Comment:
I'm surprised this isn't getting reformatted, wanna run prettier to spare the next person a big reformat diff if they have to run it?

Reply:
I though we ran prettier through eslint.  I guess not